src/content.js

```diff
@@ -310,18 +310,19 @@ export function removeFormattingElem (host, range, elem) {
   })
 }
 
-export function removeFormatting (host, range, tagName) {
+export function removeFormatting (host, range, tagName, selector) {
```

Comment:
We could shorten this to: `export function removeFormatting (host, range, selector) {`

This would still be compatible with the old version.

Reply:
Good point, thanks @peyerluk! Commit https://github.com/livingdocsIO/editable.js/pull/228/commits/56491acacdaecd2537eea8197ccafd809ca7f35a.